app/code/Magento/Wishlist/Controller/Index/Send.php

```diff
@@ -204,7 +204,7 @@ public function execute()
                 $error = __('Please enter an email address.');
             } else {
                 if (count($emails) > $emailsLeft) {
-                    $error = __('This wish list can be shared %1 more times.', $emailsLeft);
+                    $error = __('Maximum of %1 Emails can be Sent.', $emailsLeft);
```

Comment:
Please adjust the wording to `Maximum of %1 emails can be sent.` (letter casing).

Reply:
This still should be adjusted.